history/src/postgres/postgres_transaction.rs

```diff
@@ -32,6 +32,81 @@ impl From<&TransactionInfo> for PostgresTransaction {
             cu_consumed: value.cu_consumed.map(|x| x as i64),
             recent_blockhash: value.recent_blockhash.clone(),
             message: value.message.clone(),
+            slot: slot as i64,
+        }
+    }
+
+    pub fn create_statement(schema: &String) -> String {
+        format!(
+            "\
+        CREATE TABLE {}.TRANSACTIONS (
+            signature CHAR(88) NOT NULL,
```

Comment:
_CHAR_ is not recommended:

> Tip
There is no performance difference among these three types, apart from increased storage space when using the blank-padded type, and a few extra CPU cycles to check the length when storing into a length-constrained column. While character(n) has performance advantages in some other database systems, there is no such advantage in PostgreSQL; in fact character(n) is usually the slowest of the three because of its additional storage costs. In most situations text or character varying should be used instead.

https://www.postgresql.org/docs/current/datatype-character.html